src/sentry/buffer/redis.py

```diff
@@ -320,6 +328,14 @@ def _process_single_incr(self, key):
                 # TODO(dcramer): legacy pickle support - remove in Sentry 9.1
                 filters = pickle.loads(values.pop("f"))
 
+            exclude_filters_values = None
+            if "ef" in values:
+                if values["ef"].startswith("{"):
+                    exclude_filters_values = self._load_values(json.loads(values.pop("ef")))
```

Comment:
i don't think i need to do this? ^

Reply:
you can just assume its json since it hasn't existed previously